solutionbox/inception/datalab_solutions/inception/_util.py

```diff
@@ -184,3 +186,54 @@ def get_sources_from_dataset(p, dataset, mode):
     return p | 'Read source from BigQuery (%s)' % mode >> beam.io.Read(bq_source)
   else:
     raise ValueError('Invalid DataSet. Expect CsvDataSet or BigQueryDataSet')
+
+
+def decode_and_resize(image_str_tensor):
+  """Decodes jpeg string, resizes it and returns a uint8 tensor."""
+
+  # These constants are set by Inception v3's expectations.
+  height = 299
+  width = 299
```

Comment:
It's just Inception expects such dimension. Don't know the reason behind but there is already comments saying this is expected by Inception.